libs/components/src/banner/banner.component.html

```diff
@@ -1,5 +1,5 @@
 <div
-  class="tw-flex tw-items-center tw-gap-2 tw-p-2 tw-pl-4 tw-text-contrast"
+  class="tw-flex tw-items-center tw-gap-2 tw-p-2 tw-pl-4 tw-text-contrast tw-border-text-contrast tw-border-solid tw-border-b tw-border-0"
```

Comment:
You can use transparent and bg-clip padding to ensure it always matches the background color.

```suggestion
  class="tw-flex tw-items-center tw-gap-2 tw-p-2 tw-pl-4 tw-text-contrast tw-border-transparent tw-bg-clip-padding tw-border-solid tw-border-b tw-border-0"
```

Reply:
Nice